privacyidea/lib/auditmodules/sqlaudit.py

```diff
@@ -76,17 +76,32 @@ class Audit(AuditBase):
     """
     This is the SQLAudit module, which writes the audit entries
     to an SQL database table.
-    It requires the configuration parameters.
-    PI_AUDIT_SQL_URI
+    It requires the configuration parameters in pi.cfg:
+    * PI_AUDIT_KEY_PUBLIC
+    * PI_AUDIT_KEY_PRIVATE
+
+    If you want to host the SQL Audit database in another DB than the
+    token DB, you can use:
+    * PI_AUDIT_SQL_URI
+
+    It also takes the optional parameters:
+    * PI_AUDIT_POOL_SIZE
+    * PI_AUDIT_POOL_RECYCLE
+    * PI_AUDIT_SQL_TRUNCATE
+    * PI_AUDIT_NO_SIGN
+
+    You can use PI_AUDIT_NO_SIGN = True to avoid signing of the audit log.
     """
     
     def __init__(self, config=None):
         self.name = "sqlaudit"
         self.config = config or {}
         self.audit_data = {}
+        self.no_sign = self.config.get("PI_AUDIT_NO_SIGN")
```

Comment:
Personally I find having a ``no_sign`` variable a bit confusing because of the double negation :-/ What about having a ``sign_entries`` variable instead?

Or even, we could avoid introducing any new variable and just use the ``sign_object`` variable instead and set it to ``None`` if signing is turned off.

What do you think?

Reply:
I wanted to avoid relying on sign_object, because the logic could change in future  - like failing to create a sign object, although we would want to have one.

I will remove the double negation!